src/ResourceManager/Compute/Commands.Compute/Generated/Disk/Config/NewAzureRmDiskUpdateConfigCommand.cs

```diff
@@ -39,7 +41,8 @@ public partial class NewAzureRmDiskUpdateConfigCommand : Microsoft.Azure.Command
             Position = 0,
             ValueFromPipelineByPropertyName = true)]
         [Alias("AccountType")]
-        public PSStorageAccountTypes? SkuName { get; set; }
+        [PSArgumentCompleter("StandardLRS", "PremiumLRS")]
```

Comment:
Can we set "Standard_LRS" and "Premium_LRS" for completer?  Because the API takes a string value with '_', showing the exact values for completer is better.